addons/survey/static/src/js/survey_result.js

```diff
@@ -146,17 +147,19 @@ publicWidget.registry.SurveyResultChart = publicWidget.Widget.extend({
             },
             options: {
                 scales: {
-                    xAxes: [{
+                    x: {
+                        axis: "x",
                         ticks: {
                             callback: this._customTick(25),
                         },
-                    }],
-                    yAxes: [{
+                    },
+                    y: {
+                        axis: "y",
```

Comment:
some useless `axis: "x"` and `axis: "y"` all around

Reply:
Not so useless if you use it to grep all axis for chart.js